packages/esm-service-queues-app/src/send-patient-back-toqueue/send-back-patient-toqueue.resource.ts

```diff
@@ -0,0 +1,63 @@
+import { openmrsFetch, type OpenmrsResource, type Patient, type Visit } from '@openmrs/esm-framework';
```

Comment:
Should we adopt a name like 'add-patient-to-queue.resource'?

Reply:
Yeah, either "Add patient to queue" or "Transition patient" (depending on which one we intend).